python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -1288,16 +1298,21 @@ async def offset_xy(self, x, y, relative=False):
             `False` (default) offset replaces any existing offsets.
         """
 
-        self.log.debug(f"Applying x/y offset: {x}/ {y} ")
+        self.log.debug(f"Calculating x/y offset: {x}/ {y} ")
         azel = await self.rem.atmcs.tel_mount_AzEl_Encoders.aget()
-        nasmyth = await self.rem.atmcs.tel_mount_Nasmyth_Encoders.aget()
+        self.log.debug("got azel event ")
+        await asyncio.sleep(0)
+        nasmyth = await self.rem.atmcs.tel_mount_Nasmyth_Encoders.aget(timeout=1)
+        self.log.debug("got nasmyth event ")
+        await asyncio.sleep(0)
```

Comment:
You just came out of an `await` so there's really no need for this sleep here... Also, consider removing the debug message above.